### DISCUSSION TITLE
Is it possible to specify queue for chained actions

### DISCUSSION THREAD

**Discussion by Amirtheahmed** [spatie/laravel-queueable-action#71] [Q&A]
Hi spatie, Thank you for this great package. 
Quick question, How can I specify the default queue for chained actions because currently all chained actions run on the default queue by default. 
Thank you

**Comment by jacksontong** [comment]
You can use `onQueue` directly with ActionJob
```php
app(MyAction::class)
    ->onQueue()
    ->execute(...$args)
    ->chain([
        (new ActionJob(AnotherAction::class, $args))->onQueue('test-queue'),
    ]);
```
